server/dummy_mav_handler.py

```diff
@@ -14,8 +14,12 @@ def __init__(self, config, socketio):
             self.voltage = self.throttle = self.lat = self.lon = None
         self.params = None
 
-    def connect(self):
+    def connect(self, waypoints):
         print("Created dummy mav handler")
+        self.waypoints = waypoints
+        self.waypoint_index = 1 % len(waypoints)
+        self.lat = waypoints[0].latitude
+        self.lon = waypoints[0].longitude
```

Comment:
Why exactly are you setting the Dummy MAV's latitude and longitude? Coul you explain the changes made to this file in general? It seems like you are attempting to generate more accurate Dummy MAV Data.

Reply:
It makes it so the dummy plane follows the waypoints around the map. I wanted to talk to you more about architecture but you won't be at the meeting today.